app/views.py

```diff
@@ -135,32 +140,30 @@ def create_card(deck_id: int):
 
     logger.info(
         f"Handling '/deck/{deck_id}/create_card' route, user is authenticated")
-    
+
     if request.method == "POST":
 
-        logger.info(
-            f"Handling '/deck/{deck_id}/create_card' route, create_card form is submitted")
         word = request.form['word']
         translation = request.form["translation"]
 
         logger.info(
-            f"Handling '/deck/{deck_id}/create_card' route, card details word: {word}, translation: {translation}")
+            f"Handling '/deck/{deck_id}/create_card' route, create_card form is submitted. Form details word: {word}, translation: {translation}")
 
-        if CardsRepository.create(deck_id, word, translation):
-            logger.info(
-                f"Handling '/deck/{deck_id}/create_card' route, card is created and saved into database")
+        if not CardsRepository.create(deck_id, word, translation):
+            logger.error(
+                f"Handling '/deck/{deck_id}/create_card' route, card has been not created")
             logger.info(
                 f"Handling '/deck/{deck_id}/create_card' route, rendering response")
-            return "<h2>New card has been created</h2>"
+
+            return "<h2>There was some error</h2>"
 
         logger.info(
-            f"Handling '/deck/{deck_id}/create_card' route, card has been not created")
+            f"Handling '/deck/{deck_id}/create_card' route, card is createde")
```

Comment:
typo createde
and before you were writing something like 'was created succesfully', keep same format

Reply:
done